upgrade/upgrade.go

```diff
@@ -95,6 +96,9 @@ func upgrade(currentNodeID, namespace string, config *restclient.Config, lhClien
 				if err := doInstanceManagerUpgrade(namespace, lhClient); err != nil {
 					logrus.Errorf("cannot finish InstanceManager upgrade: %v", err)
 				}
+				if err := doNodeUpgrade(namespace, lhClient, kubeClient); err != nil {
```

Comment:
I thought we're going to do it in the node controller. We already have the Kubernetes node information in the node controller. We can simply update the information there without handle it specifically using CreateDefaultNode or upgrade path.